packages/next/src/client/components/react-dev-overlay/server/middleware.ts

```diff
@@ -333,32 +330,30 @@ function getOverlayMiddleware(options: OverlayMiddlewareOptions) {
       )
 
       let source: Source = null
-      let sourcePackage: string | undefined = undefined
+      const sourceModule = findCallStackFrameModule(moduleId)
       const clientCompilation = options.stats()?.compilation
       const serverCompilation = options.serverStats()?.compilation
       const edgeCompilation = options.edgeServerStats()?.compilation
       const isFile = frame.file.startsWith('file:')
 
+      // REVIEW: Source always null
```

Comment:
I don't know why, but `getSourceById` always returns `null`, so sending `sourceModule` from the server stopped working. To proceed, I moved this logic to the client for now.